src/coord/Coord.cc

```diff
@@ -68,14 +66,20 @@ CoordSystemMap const getCoordSystemMap() {
     return idMap;
 }
 
-/// Calculate angular distance between two points using the Haversine formula
-///
-/// Besides the differences between the two coordinates, we also input the
-/// cosine of the two declinations, so as to be thrifty with the use of trig functions.
-afwGeom::Angle haversine(afwGeom::Angle const dAlpha, /// Difference between RAs (or longitudes), alpha1-alpha2
-                         afwGeom::Angle const dDelta, /// Difference between Decs (or latitudes), delta1-delta2
-                         double const cosDelta1, /// Cosine of the first Dec, cos(delta1)
-                         double const cosDelta2 /// Cosine of the second Dec, cos(delta2)
+/** Calculate angular distance between two points using the Haversine formula
+ *
+ * Besides the differences between the two coordinates, we also input the
+ * cosine of the two declinations, so as to be thrifty with the use of trig functions.
+ *
+ * @param dAlpha Difference between RAs (or longitudes), alpha1-alpha2
+ * @param dDelta Difference between Decs (or latitudes), delta1-delta2
+ * @param cosDelta1 Cosine of the first Dec, cos(delta1)
+ * @param cosDelta2 Cosine of the second Dec, cos(delta2)
+ */
+afwGeom::Angle haversine(afwGeom::Angle const dAlpha,
+                         afwGeom::Angle const dDelta,
+                         double const cosDelta1,
+                         double const cosDelta2
```

Comment:
Per the current style guide, public *and protected* functions must be documented. :wink:

I'm willing to add `@internal` for documentation of private/local code, but I do think trusting in the centralized settings is the better long-term solution. Perhaps @jonathansick can weigh in on how this decision affects DMTN-030-type plans?